### DISCUSSION TITLE
Overrating of elemental dmg on weapons

### DISCUSSION THREAD

**Discussion by Borega** [isid0re/SoloLeveling#129] [Ideas]
currently watching my hammerdin level on level 78 and he prioritises elemental dmg way to much. He changed a spirit for a sword with 380 light dmg. 
At least for casters that should be changed

**Comment by theBGuy** [comment]
Could you post the tier of the sword? Casters should rate elemental damage as 0

**Reply by Borega** [reply]
Its a normal blade

**Comment by theBGuy** [comment]
Oh get out of here with using this for pod or pd2

Sent via the Samsung Galaxy S9, an AT&T 5G Evolution capable smartphone
Get Outlook for Android<https://aka.ms/ghei36>

________________________________
